rosdep/python.yaml

```diff
@@ -8479,6 +8479,17 @@ python3-pytest-xdist:
     "*": [python3-pytest-xdist]
     "7": ['python%{python3_pkgversion}-pytest-xdist']
   ubuntu: [python3-pytest-xdist]
+python3-pytest-xvfb:
+  arch: [python-pytest-xvfb]
+  debian: [python3-pytest-xvfb]
+  fedora: [python3-pytest-xvfb]
+  gentoo: [dev-python/pytest-xvfb]
+  nixos: [python312Packages.pytest-xvfb]
+  opensuse: [python3-pytest-xvfb]
+  rhel:
+    pip:
```

Comment:
> Standard pip disclaimer: ROS packages that depend on pip keys cannot be released into a ROS distribution. They can only be depended on by from-source builds. Because of this, system packages are highly preferred to pip packages.